tests/wpt/meta/content-security-policy/nonce-hiding/script-nonces-hidden-meta.sub.html.ini

```diff
@@ -1,6 +1,3 @@
 [script-nonces-hidden-meta.sub.html]
-  [Writing 'nonce' IDL attribute.]
-    expected: FAIL
-
   [createElement.nonce.]
```

Comment:
I am a bit confused why this one isn't passing, but it's been a long evening. So need to investigate later what's going on.

Reply:
It has to do with `document.currentScript` pointing to the wrong value. I think this is not a problem with handling nonces, but a problem in the JS engine. Therefore, let's leave this test failing for now until the JS engine updates are there.